main/main.go

```diff
@@ -49,6 +49,12 @@ func main() {
 		fmt.Printf("couldn't build config: %s\n", err)
 		os.Exit(1)
 	}
+	// Initialize the Warp Quorum values by fetching via RPC
+	// We do this here so that BuildConfig doesn't need to make RPC calls
+	if err = cfg.InitializeWarpQuorum(); err != nil {
+		fmt.Printf("couldn't initialize warp quorum: %s\n", err)
+		os.Exit(1)
```

Comment:
I don't think there's any reason to prefer and exit of a panic here. We currently basically get the same behaviour between panic and exit, but if any of this code becomes encapsulated, or any defer statements are made before any of these exit points, it'll cause problems.

A panic causing the program to exit will print `exit status 2`, which is go's exit status, but it actually returns `1` as the exit code to the system, so we get the same behaviour there.

I think it's a good idea to always prefer using the `%w` verb to wrap errors. Again, it doesn't really matter for this _specific_ code, but it's a good practice so that the type of the error is preserved for `errors.Is` and `errors.As`.

Reply:
Great points! I've gone ahead and changed all the returns with panics in the `main` function.